src/video_core/renderer_vulkan/vk_rasterizer.cpp

```diff
@@ -238,7 +238,7 @@ void Rasterizer::Draw(bool is_indexed, u32 index_offset) {
                            instance_offset);
     } else {
         const u32 num_vertices =
-            regs.primitive_type == AmdGpu::PrimitiveType::RectList ? 4 : regs.num_indices;
+            regs.primitive_type == AmdGpu::PrimitiveType::RectList ? 3 : regs.num_indices;
```

Comment:
Is this supposed to still be changing the `num_vertices`? Thought this was supposed to support proper number of vertices e.g. multiple rectangles.

Reply:
This is debugging leftover yes, can just use regs.num_indices